gallery/how_to/compile_models/from_onnx.py

```diff
@@ -21,13 +21,12 @@
 
 This article is an introductory tutorial to deploy ONNX models with Relay.
 
-For us to begin with, ONNX package must be installed.
-
-A quick solution is to install protobuf compiler, and
+For us to begin with, ONNX package must be installed:
```

Comment:
nit: match the wording update of the other files:

```suggestion
To begin, install the ONNX package:
```
or something like that..